src/styles/primeng-theme/components/_splitbutton.scss

```diff
@@ -30,4 +30,49 @@ p-splitbutton {
             padding-right: 0;
         }
     }
+}
+
+p-splitbutton[iconButtonOptions] {
```

Comment:
iconButtonOptions it's way too long, just put it as "icon" because it's attached to the ```p-splitbutton``` tag, also you can nest it inside the first ```p-splitbutton``` in this file and use ```&```